website/site/content/docs/widgets/list.md

```diff
@@ -8,14 +8,19 @@ type: "widget"
 
```

Comment:
The only changes this file should be additions - changes like switching the list item marker from `*` to `-` are code style changes.